src/platform/packages/shared/shared-ux/modal/tabbed/src/tabbed_modal.tsx

```diff
@@ -173,7 +173,9 @@ const TabbedModalInner: FC<ITabbedModalInner> = ({
           <EuiTabs>{renderTabs()}</EuiTabs>
           <EuiSpacer size="m" />
           {React.createElement(function RenderSelectedTabContent() {
-            useLayoutEffect(onTabContentRender, []);
+            useLayoutEffect(() => {
```

Comment:
@eokoneyo, thanks for the flaky fix 🙇 just curious this isn't a plain useEffect instead?